pkg/kubernetes/kubectl.go

```diff
@@ -0,0 +1,266 @@
+	pingKafka := []string{"run", "-n", namespace, "kafka-ping", "-ti", "--rm=true", "--image=strimzi/kafka:0.20.0-kafka-2.6.0", "--restart=Never", "--", "bin/kafka-topics.sh", "--list", "--bootstrap-server=my-cluster-kafka-external-bootstrap:9094"}
+	log.Print(pingKafka)
+	return helm.RunCmdWithTimeout(ctx, "kubectl", pingKafka)
+}
+func (kubectl kubernetesClient) Insert(ctx context.Context, topic, namespace string) error {
+	err := produce(ctx, topic, namespace)
+	return err
+}
+func (kubectl kubernetesClient) Count(ctx context.Context, topic, namespace string) (int, error) {
+	count, err := consume(ctx, topic, namespace)
+	return count, err
+}
+
+// to produce messages to kafka
+func produce(ctx context.Context, topic string, namespace string) error {
+	partition := 0
+	forwarder, err := K8SServicePortForward(ctx, "my-cluster-kafka-external-bootstrap", namespace, "9094")
+	if err != nil {
+		return err
+	}
+
+	defer forwarder.Close()
+	ports, err := forwarder.GetPorts()
+	if err != nil {
+		log.Print("failed to get port:", err)
+		return err
+	}
+	uri := "localhost:" + fmt.Sprint(ports[0].Local)
+
+	conn, err := kafka.DialLeader(ctx, "tcp", uri, topic, partition)
+	if err != nil {
+		return err
+	}
+	err = conn.SetWriteDeadline(time.Now().Add(10 * time.Second))
+	if err != nil {
+		return err
+	}
+	_, err = conn.WriteMessages(
+		kafka.Message{Value: []byte("{'title':'The Matrix','year':1999,'cast':['Keanu Reeves','Laurence Fishburne','Carrie-Anne Moss','Hugo Weaving','Joe Pantoliano'],'genres':['Science Fiction']}")},
+	)
+	if err != nil {
+		return err
+	}
+
+	if err := conn.Close(); err != nil {
+		log.Print("failed to close connection:", err)
+		return err
+	}
+	return nil
+}
+
+// LoadConfig returns a kubernetes client config based on global settings.
+func LoadConfig() (*rest.Config, error) {
+	//	log.Print("Attempting to use InCluster config")
+	config, err := rest.InClusterConfig()
+	if err == nil {
+		return config, nil
+	}
+	//	log.Print("Attempt to use InCluster config failed")
+
+	homeConfig := filepath.Join(os.Getenv("HOME"), ".kube/config")
+	config, err = clientcmd.BuildConfigFromFlags("", homeConfig)
+	if err == nil {
+		return config, nil
+	}
+	return nil, errors.Wrapf(err, "Failed to load default k8s config")
+}
+
+// K8SServicePortForward creates a service port forwarding and returns the forwarder and the error if any
+func K8SServicePortForward(ctx context.Context, svcName string, ns string, pPort string) (*portforward.PortForwarder, error) {
+	var selector string
+	errCh := make(chan error)
+	readyChan := make(chan struct{}, 1)
+
+	cfg, err := LoadConfig()
+	if err != nil {
+		panic(err.Error())
+	}
+
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		panic(err.Error())
+	}
+	roundTripper, upgrader, err := spdy.RoundTripperFor(cfg)
+	if err != nil {
+		return &portforward.PortForwarder{}, errors.Wrapf(err, "Failed to create RoundTripper for k8s config")
```

Comment:
```suggestion
		return nil, errors.Wrapf(err, "Failed to create RoundTripper for k8s config")
```
Please return nil in case of error at all the occurrences